caldav/objects.py

```diff
@@ -47,6 +47,29 @@ def errmsg(r):
     return "%s %s\n\n%s" % (r.status, r.reason, r.raw)
 
 
+def _expand_event(event, start, end):
+    """
+    :param event: Event
+    """
+    import icalendar
+    import recurring_ical_events
```

Comment:
We already have a dependency on `dateutil.rrule`.  If it's easy to rewrite this code to also use `dateutil.rrule`, or if it's easy to rewrite my code to use `recurring_ical_events`, then I think it would be nice to reduce the number of dependencies.  If not, I think we can move this import also to the top of the file, and add it into `setup.py`.

Reply:
Tests also break because the dependency is not mirrored in `setup.py`

Reply:
> If it's easy to rewrite this code to also use `dateutil.rrule`, or if it's easy to rewrite my code to use `recurring_ical_events`

Ok I'll look into it.